ludwig/__init__.py

```diff
@@ -13,4 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
+import sys
+import logging
+
 from ludwig.globals import LUDWIG_VERSION as __version__
+
+
+logging.basicConfig(level=logging.INFO, stream=sys.stdout, format='%(message)s')
```

Comment:
You should configure the top-level ludwig logger instead of the root level logger, this way setting info or debug won't activate info or debug logs for other libraries. The gist of `basicConfig` should be quite easy to extract for your need: https://github.com/python/cpython/blob/master/Lib/logging/__init__.py#L1893

Reply:
Yeah, makes sense. Will update this as well.